tcms/templates/case/edit.html

```diff
@@ -1,178 +0,0 @@
-{% extends "tcms_base.html" %}
-{% load i18n %}
-{% load static %}
-{% block subtitle %}
-    {% if test_case %}
-        {% trans "Edit test case" %} - {{ test_case.summary }}
-    {% else %}
-        {% trans "Add new test case" %}
-    {% endif %}
-{% endblock %}
-
-{% block extra_head %}
-{{ form.media }}
-{% endblock %}
-
-{% block custom_stylesheet %}
-<style type="text/css">
-#testcase_cc_list {
-	margin-top: 3px;
-}
-
-#id_cc_list {
-	width: 100%;
-}
-</style>
-{% endblock %}
-
-{% block custom_javascript %}
-<script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
-<script src="{% static 'js/jsonrpc.js' %}"></script>
-<script src="{% static 'js/utils.js' %}"></script>
-
-<script type="text/javascript" src="{% static 'js/testcase_actions.js' %}"></script>
-<script type="text/javascript">
-Nitrate.Utils.after_page_load(Nitrate.TestCases.Edit.on_load);
```

Comment:
This comes along with JavaScrit which we also don't need. Once removed there will probably be even more JS functions which can be removed.

Reply:
Unfortunately, not that much code was removed